src/FluentValidation.DependencyInjectionExtensions/FluentValidation.DependencyInjectionExtensions.csproj

```diff
@@ -1,40 +1,40 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Latest commit seems to have reformatted the project file - can this be reverted? It makes it hard to review changes